checkov/common/checks_infra/solvers/attribute_solvers/__init__.py

```diff
@@ -40,3 +40,4 @@
 from checkov.common.checks_infra.solvers.attribute_solvers.number_of_words_greater_than_or_equal_attribute_solver import NumberOfWordsGreaterThanOrEqualAttributeSolver  # noqa
 from checkov.common.checks_infra.solvers.attribute_solvers.number_of_words_less_than_or_equal_attribute_solver import NumberOfWordsLessThanOrEqualAttributeSolver  # noqa
 from checkov.common.checks_infra.solvers.attribute_solvers.number_of_words_less_than_attribute_solver import NumberOfWordsLessThanAttributeSolver  # noqa
+from checkov.common.checks_infra.solvers.attribute_solvers.not_within_attribute_solver import NotWithinAttributeSolver  # noqa
```

Comment:
also this one, just move it under `WithinAttributeSolver`

Reply:
done